docs/pages/eas/hosting/reference/responses-and-headers.mdx

```diff
@@ -64,15 +64,50 @@ EAS Hosting will respond with an error page in these cases. The error page will
 
 ## Request headers
 
-**EAS Hosting** will add a couple of headers to every request, before it's forwarded to your API routes. These headers generally add more information about who made the request.
+**EAS Hosting** will add a couple of headers to every request, before they're forwarded to your API routes. These headers generally add more information about who made the request.
```

Comment:
```suggestion
**EAS Hosting** will add the following headers to every request, before they're forwarded to your API routes. These headers generally add more information about who made the request.
```

Considering the list of headers below, it seems there are more than 2 headers added to each request now.